src/site/apt/building/cmake.apt

```diff
@@ -39,24 +39,26 @@ $ sudo make install
-| -DAPR_STATIC=yes       | Link to the APR static library. By default, the log4cxx shared library is linked to the     |
-|                        | APR shared library. If BUILD_SHARED_LIBS=off, the static APR library is always used.        |
-*------------------------+---------------------------------------------------------------------------------------------+
+*-----------------------------------+---------------------------------------------------------------------------------------------+
+| -DLOG4CXX_WCHAR_T=no              | Enable wchar_t API methods, choice of yes (default), no.                                    |
+*-----------------------------------+---------------------------------------------------------------------------------------------+
+| -DLOG4CXX_UNICHAR=yes             | Enable UniChar API methods, choice of yes, no (default).                                    |
+*-----------------------------------+---------------------------------------------------------------------------------------------+
+| -DLOG4CXX_CFSTRING=yes            | Enable CFString API methods, requires Mac OS/X CoreFoundation, choice of yes, no (default). |
+*-----------------------------------+---------------------------------------------------------------------------------------------+
+| -DBUILD_TESTING=off               | Do not build tests.  Tests are built by default                                             |
+*-----------------------------------+---------------------------------------------------------------------------------------------+
+| -DBUILD_SHARED_LIBS=off           | Build log4cxx as a static library. A dynamically linked log4cxx library is built by default.|
+|                                   | Any compilation unit that includes a log4cxx header must define LOG4CXX_STATIC.             |
+*-----------------------------------+---------------------------------------------------------------------------------------------+
+| -DAPU_STATIC=yes                  | Link to the APR-Util static library. By default, the log4cxx shared library is linked to the|
+|                                   | APR-Util shared library. If BUILD_SHARED_LIBS=off, the static APR-Util library is used.     |
+*-----------------------------------+---------------------------------------------------------------------------------------------+
+| -DAPR_STATIC=yes                  | Link to the APR static library. By default, the log4cxx shared library is linked to the     |
+|                                   | APR shared library. If BUILD_SHARED_LIBS=off, the static APR library is always used.        |
+*-----------------------------------+---------------------------------------------------------------------------------------------+
+| -DEXPORT_ONLY_LOG4CXX_SYMBOLS=yes | Only export the log4cxx symbols in the log4cxx shared library.                              |
```

Comment:
Maybe I'm wrong, but the other docs seem to document the non-default values. So this should be `no` instead? Things should made be clear in the text description as well:

> -DEXPORT_ONLY_LOG4CXX_SYMBOLS=no
> Only export the log4cxx symbols in the log4cxx shared library, choice of yes (default), no.

Reply:
Updated the documentation to show the default (no).